api/revoke.go

```diff
@@ -96,7 +96,7 @@ func (h *caHandler) Revoke(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := h.Authority.Revoke(ctx, opts); err != nil {
-		WriteError(w, errs.ForbiddenErr(err))
+		WriteError(w, errs.ForbiddenErr(err, "error revoking certificate"))
```

Comment:
Maybe we should be trying to get the message from the downstream method? This isn't very useful from a user's standpoint, right?

Reply:
I don't want to return unknown errors messages, so, unless `err` is an `errs.Error`, this code will return `error revoking certificate` because we don't know anything else, the logs will show the reason by the way. 

We'll change this to a 500 later.